examples/osgi-test-example-bnd/org.osgi.test.example.player.impl/bnd.bnd

```diff
@@ -0,0 +1,7 @@
+# Build dependencies 
+-buildpath: \
+	osgi.annotation,\
+	org.osgi.test.example.api;version=latest,\
+	osgi.cmpn
```

Comment:
Don't use osgi.cmpn. It is now an anti-practice and we don't want to show using it in an example. You need to use the individual spec API jars as needed.

Reply:
removed cmpn also from the repo file